policybot/README.md

```diff
@@ -96,14 +96,14 @@ https://bots.istio.io.
 
 The bot's behavior is controlled entirely through its configuration file. The
 format of the configuration file is described by the `Args` struct in
-`pkg/config/args.go` 
+`pkg/config/args.go`.
 
 ## Deployment
 
 Here's how the bot is currently deployed:
 
 - The bot executes using Google Cloud Run. You can do a full build and deploy
-a new revision of the bot to Google Cloud Run using `make deploy`. In order for this
+a new revision of the bot to Google Cloud Run using `export GO111MODULE=on && make deploy`. In order for this
```

Comment:
Given the change to the makefile, is this export needed?

Reply:
removed.